common/db_store/error.py

```diff
@@ -0,0 +1,20 @@
+class DBError(Exception):
+
+    def __init__(self, message, rc=1, *args):
+        """Handle Exception in DB store."""
+        self._rc = rc
+        self._desc = message % (args)
+
+    @property
+    def rc(self):
+        return self._rc
+
+    @property
+    def desc(self):
+        return self._desc
+
+    def __str__(self):
+        """String representation of the object."""
```

Comment:
check function doc format
same for all function doc

Reply:
A one-liner docstring is sufficient to describe its behavior so I've added one-liner docstring.

Reply:
OK, if it is as per google format, for consistency purposes please check it once.